### PR TITLE
net-fs/samba: samba netdb defines

### PR DESCRIPTION
**Pull request by listout**
NETDB_INTERNAL and NETDB_INTERNAL are not defined in musl as a result the build is failing.
    
The patch is already implemented in ::musl, since the plan for GSoC is to kill ::musl hence moving the patch here.

### CODE REVIEW

File: net-fs/samba/samba-4.16.1.ebuild [hunk 1]
```diff
@@ -141,6 +141,7 @@ BDEPEND="${PYTHON_DEPS}
 
 PATCHES=(
 	"${FILESDIR}/${PN}-4.4.0-pam.patch"
+	"${FILESDIR}/${PN}-${PV}-netdb-defines.patch"
```
Comment (thesamesam) [inline net-fs/samba/samba-4.16.1.ebuild:144]: ${PN}-${PV} == ${P}, but you should probably just do ${PN}-4.16.1 to match existing style.

File: net-fs/samba/files/samba-4.16.1-netdb-defines.patch [hunk 2]
```diff
@@ -0,0 +1,19 @@
+diff --git a/nsswitch/wins.c b/nsswitch/wins.c
```
Comment (thesamesam) [inline net-fs/samba/files/samba-4.16.1-netdb-defines.patch:1]: 1. Run `iwdevtools` scrub-patch on this file
2. Add link to any relevant Gentoo bugs to the top of the patch (and tag them in the commit)

**Review by thesamesam — COMMENTED** [review]
please squash commits too

**Comment by listout** [comment]
@thesamesam By mistake, I pushed something that closed the issue. What can I do now, open a new one or this one can be reopend

**Pull request closed without merging**